docs/finding_legistar_id.md

```diff
@@ -0,0 +1,31 @@
+# Finding Your Municipality’s Legistar ID
+
+Legistar is the most widely used legislation tracking tool in the nation. We are 
+leveraging its wide coverage to help standardize and streamline part of our onboarding 
+process by creating generalized Legistar scrapers so you can potentially skip the step 
+of creating an event scraper. Unfortunately a municipality’s Legistar ID isn’t always 
+easily discoverable so here are some steps to find it.
+
+1. Go to your municipality’s council page and locate the meeting agendas.
+
+2. Look at the URL of the website and if it contains “legistar”, then your 
+municipality uses Legistar. If it does not, your municipality does not use Legistar.  
```

Comment:
Instead of `your municipality does not use Legistar` can we say `_likely_ does not use Legistar`

Reply:
This makes three more relevant

Reply:
"likely does not use Legistar and you should review step three for another method for determining if your municipality utilizes Legistar." (which maybe we split into 2a and 2b)??